cdap-data-fabric/src/main/java/co/cask/cdap/data2/admin/DefaultAdmin.java

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package co.cask.cdap.internal.app.runtime;
+package co.cask.cdap.data2.admin;
```

Comment:
Tests in ``cdap-data-fabric``, ``cdap-notifications``, `` cdap-watchdog`` need it. Although now I think that I should not be moving production code because its not available to tests. Any suggestions on how to fix it?